django/test/runner.py

```diff
@@ -630,10 +630,10 @@ def get_test_runner_kwargs(self):
             'buffer': self.buffer,
         }
 
-    def run_checks(self):
+    def run_checks(self, databases):
         # Checks are run after database creation since some checks require
         # database access.
-        call_command('check', verbosity=self.verbosity)
+        call_command('check', verbosity=self.verbosity, databases=databases)
```

Comment:
This lets the test command only run db checks on selected databases, should this be made more explicit in the commit message?

Reply:
I added release notes.